frontend/src/helpers/fetchMapValue.ts

```
@@ -0,0 +1,4 @@
+export function fetchMapValue<K, V>(map: Map<K, V>, key: K, fetch: () => V): V {
+  if (!map.has(key)) map.set(key, fetch());
+  return map.get(key)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)